BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -97,7 +99,20 @@ private void updateLocation() {
         return usableBeacons;
     }
 
+    public static Location walkingSpeedFilter(Location oldLocation, Location newLocation) {
```

Comment:
This doesn't belong here. Should probably be part of `DistanceUtil`.

Reply:
Create a method that is more generic (e.g. accepts a maximum or minimum speed / distance)